debian/control

```diff
@@ -200,6 +201,53 @@ Description: manager for the ceph distributed storage system
  This package contains the manager daemon, which is used to expose high
  level management and monitoring functionality.
 
+Package: ceph-mgr-diskprediction-local
+Architecture: any
+Depends: ceph-mgr (= ${binary:Version}),
+         python-numpy,
+         python-scipy,
+         python-sklearn,
+         ${misc:Depends},
+         ${python:Depends},
+         ${shlibs:Depends},
+Description: diskprediction-local plugin for ceph-mgr
+ Ceph is a massively scalable, open-source, distributed
+ storage system that runs on commodity hardware and delivers object,
+ block and file system storage.
+ .
+ This package contains the diskprediction_local plugin for the ceph-mgr
+ daemon, which helps predict disk failures.
+
+Package: ceph-mgr-diskprediction-cloud
+Architecture: any
+Depends: ceph-mgr (= ${binary:Version}),
+         ${misc:Depends},
+         ${python:Depends},
+         ${shlibs:Depends},
+Description: diskprediction-cloud plugin for ceph-mgr
+ Ceph is a massively scalable, open-source, distributed
+ storage system that runs on commodity hardware and delivers object,
+ block and file system storage.
+ .
+ This package contains the diskprediction_cloud plugin for the ceph-mgr
+ daemon, which helps predict disk failures.
+
+Package: ceph-mgr-rook
+Architecture: any
+Depends: ceph-mgr (= ${binary:Version}),
+	 python-six,
+         ${misc:Depends},
+         ${python:Depends},
+         ${shlibs:Depends},
```

Comment:
The epel7 python-kubernetes module has made its way out of testing now, so we should be good to go on adding it as a dependency here now (and I think we probably should).

Reply:
well, it's still not in Ubuntu xenial (even backports), so that's an issue for our testing.  I don't know if we plan to still support xenial in nautilus?

Reply:
Doh, got mixed up and meant to make this comment in the context of ceph.spec.in. I think it's OK to add it back into that file now.